### PR TITLE
quo2 core 

### PR DESCRIPTION
**Pull request by flexsurfer**
final cleaning
image

and introducing core namespace

### CODE REVIEW

File: src/status_im/ui/screens/communities/community_overview_redesign.cljs [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
              [status-im.ui.screens.communities.request-to-join-bottom-sheet-redesign :as request-to-join]
              [status-im.ui.screens.communities.community-options-bottom-sheet :as options-menu]
              [status-im.ui.components.list.views :as list]
              [status-im.utils.handlers :refer [<sub >evt]]
-             [status-im.ui.screens.communities.styles :as styles]
-             [quo2.foundations.colors :as colors]
-             [quo2.components.navigation.page-nav :as page-nav]
              [status-im.ui.screens.communities.icon :as communities.icon]))
-
+(def a 1)
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/communities/community_overview_redesign.cljs]: ?

**Review by Parveshdhull — APPROVED** [review]
great job 🎉

File: src/quo2/components/community/community_card_view.cljs [hunk 2]
```diff
@@ -1,48 +1,44 @@
 (ns quo2.components.community.community-card-view
   (:require
    [quo2.components.community.community-view :as community-view]
-   [status-im.utils.handlers :refer [<sub]]
-   [status-im.ui.components.react :as react]
-   [status-im.ui.screens.communities.styles :as styles]
-   [status-im.ui.screens.communities.icon :as communities.icon]))
+   [quo2.components.community.styles :as styles]
```
Comment (OmarBasem) [inline src/quo2/components/community/community_card_view.cljs]: I think we agreed on using `style` and not `styles`?

File: src/quo2/core.cljs [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+(def information-box quo2.components.info.information-box/information-box)
+(def gap quo2.components.messages.gap/gap)
+(def system-message quo2.components.messages.system-message/system-message)
+(def reaction quo2.components.reactions.reaction/reaction)
+(def tags quo2.components.tags.tags/tags)
+(def tabs quo2.components.tabs.tabs/tabs)
+(def scrollable-tabs quo2.components.tabs.tabs/scrollable-tabs)
+(def account-selector quo2.components.tabs.account-selector/account-selector)
+
+;; AVATAR
```
Comment (ilmotta) [inline src/quo2/core.cljs]: Just a tiny change, in Lisps it's more idiomatic to use 3 or 4 semicolons for section-like comments. Some editors (e.g. Emacs with `outline-mode`) will take advantage of it in many useful ways. This is also recommended in the Clojure Style Guide.

File: src/status_im/ui/screens/communities/community_overview_redesign.cljs (hunk 1 above)
Comment (OmarBasem) [inline src/status_im/ui/screens/communities/community_overview_redesign.cljs]: What is this for?

**Review by ilmotta — APPROVED** [review]

**Review by OmarBasem — APPROVED** [review]

File: src/status_im/ui/screens/communities/community_overview_redesign.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/communities/community_overview_redesign.cljs]: for hot reload :) leftover

**Pull request merged**